synapse/api/ratelimiting.py

```diff
@@ -1,4 +1,5 @@
 # Copyright 2014-2016 OpenMarket Ltd
+# Copyright 2020 The Matrix.org Foundation C.I.C.
```

Comment:
Do we normally update copyrights like this?

Reply:
Yes, typically, though it's pretty much optional.